Products/GenericSetup/registry.py

```diff
@@ -644,6 +644,10 @@ def __init__(self):
     def getProfileInfo(self, profile_id, for_=None):
         """ See IProfileRegistry.
         """
+        if profile_id.startswith("profile-"):
+            profile_id = profile_id[len('profile-'):]
```

Comment:
Minor quibble but what's with `len('profile-')`? `profile_id[8:]` would make more sense to me as this is not dynamic.